CONTRIBUTING.md

```diff
@@ -8,9 +8,8 @@ Thanks for being willing to contribute!
 ## Project setup
 
 1. Fork and clone the repo
-2. `$ npm install` to install dependencies
-3. `$ npm run validate` to validate you've got it working
-4. Create a branch for your PR
+2. `npm run setup --silent` to setup the project (it installs deps and runs the validate script)
```

Comment:
I don't think the `--silent` is needed here, that's already been added to the validate command under the setup script.

Reply:
Nope, still need it to prevent npm from logging useless stuff when running the command I'm afraid.